README.md

```diff
@@ -1,2 +1,8 @@
 # code-genie-bot
 telegram bot for code genie
+
+# Environment Variables
+
```

Comment:
explain also that these vars should be stored inside bashrc, with the exact keys, and in what way will they be loaded into a python file (i.e os modules, which functions etc, in short)